src/main/java/fi/otavanopisto/kuntaapi/server/controllers/IdentifierController.java

```diff
@@ -280,21 +301,57 @@ public Long getIdentifierOrderIndex(String kuntaApiId) {
  
     return result;
   }
+  
+  private ArchivedIdentifier findArchivedIdentifierById(BaseId id) {
+    if (id == null) {
+      return null;
+    }
+    
+    String organizationKuntaApiId = null;
+    if (id instanceof OrganizationBaseId) {
+      organizationKuntaApiId = getOrganizationBaseIdOrganizationKuntaApiId((OrganizationBaseId) id);
+    }
+    
+    return findArchivedIdentifierByTypeSourceIdAndOrganizationId(id.getType().name(), id.getSource(), id.getId(), organizationKuntaApiId);
+  }
+  
+  private ArchivedIdentifier findArchivedIdentifierByTypeSourceIdAndOrganizationId(String type, String source, String sourceId, String organizationKuntaApiId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "findArchivedIdentifierByTypeSourceIdAndOrganizationId" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)